charts/ceph-csi-cephfs/README.md

```diff
@@ -6,22 +6,46 @@ The ceph-csi-cephfs chart adds cephfs volume support to your cluster.
 
 To install the Chart into your Kubernetes cluster
 
+- For helm 2.x
+
 ```bash
```

Comment:
By adding the `-` you just created a list that requires indention. Make sure to indent the the `\```bash` and lines below.

I suggest to use hackmd.io for checking the layout.